usagov/usagov-2021
# Usagov 2073 published pages

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2073

## Description

Moves generation of the published pages CSV out of the tome export to a drush command that can be executed once the export completes.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- On dev branch, run tome export and save published-pages.csv as published-pages-good.csv
- Switch to this branch, run tome export. Compare generated published-pages.csv to good branch above. I do this by sorting the files so they have things in the same order (by path and then node id should work), then using a diff tool to compare them.

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
